integration/query_compat_test.go

```diff
@@ -115,6 +110,8 @@ func testQueryCompat(t *testing.T, testCases map[string]queryCompatTestCase) {
 }
 
 func TestQueryCompat(t *testing.T) {
+	setup.SkipForTigrisWithReason(t, "https://github.com/FerretDB/FerretDB/issues/908")
```

Comment:
Ok, but why skip that one?

Reply:
yes true...
thanks for catching this one.